wgpu/src/api/instance.rs

```diff
@@ -395,4 +412,12 @@ impl Instance {
     pub fn generate_report(&self) -> Option<wgc::global::GlobalReport> {
         self.inner.as_core_opt().map(|ctx| ctx.generate_report())
     }
+
+    /// Returns set of supported WGSL language extensions supported by this instance
```

Comment:
typo: Missing period.

```suggestion
    /// Returns set of supported WGSL language extensions supported by this instance.
```